tools/build_config/build.ps1

```diff
@@ -139,6 +130,7 @@ function New-CMake-Generator-Command {
 
 function Write-Versions {
   Write-Output "$(cmake --version)"
+  Write-Output "Matlab: $($(Get-Command matlab.exe).Source)"
```

Comment:
Following our discussions on Slack, should we be printing this? The Matlab on the path may not be the Matlab that CMake picks up (since it searches the registry). This is a difference to Linux, where CMake will pick whichever Matlab is on the path

Reply:
why isn't this the correct path? how does CTest run with the correct MATLAB if the console can't find it? How is that path defined??

Reply:
By default, at configure time, CMake will pick the latest version of Matlab out of the registry. All the data needed to run CTest is generated in the CMake configure stage, including the path to the Matlab it found. Have a look inside `build/_test/CTestTestfile.cmake`, it contains the full test commands, including the absolute path to the Matlab it found at configure time.

We can specify what version of Matlab CMake looks for by calling `find_package(Matlab EXACT ${_version})` - this is done in `PACE_FindMatlab` in Herbert.

Reply:
I'm not sure we need any of them besides the cmake version as they're reported in the CMake configure

Reply:
I think I agree. We should at least remove the Matlab one as it is potentially misleading